src/redux/actions/login-logout.js

```diff
@@ -0,0 +1,5 @@
+export const LOGIN = 'LOGIN';
```

Comment:
We have to make a separate directory for action types. All the constants will be defined there ..

Reply:
Alright, Working on it.

Reply:
Resolved